core/src/main/scala/chisel3/Data.scala

```diff
@@ -257,95 +257,25 @@ object chiselTypeOf {
   * Thus, an error will be thrown if these are used on bound Data
   */
 object Input {
-  def apply[T <: Data](source: => T): T = {
+  def apply[T <: IsBindable](source: => T): T = {
     SpecifiedDirection.specifiedDirection(source)(_ => SpecifiedDirection.Input)
   }
 }
 object Output {
-  def apply[T <: Data](source: => T): T = {
+  def apply[T <: IsBindable](source: => T): T = {
     SpecifiedDirection.specifiedDirection(source)(_ => SpecifiedDirection.Output)
   }
 }
 
 object Flipped {
-  def apply[T <: Data](source: => T): T = {
+  def apply[T <: IsBindable](source: => T): T = {
     SpecifiedDirection.specifiedDirection(source)(x => SpecifiedDirection.flip(x.specifiedDirection))
   }
 }
 
-/** This forms the root of the type system for wire data types. The data value
-  * must be representable as some number (need not be known at Chisel compile
-  * time) of bits, and must have methods to pack / unpack structured data to /
-  * from bits.
-  *
-  * @groupdesc Connect Utilities for connecting hardware components
-  * @define coll data
+/** This forms the roots of the type system for wire and non-wire data types.
```

Comment:
I think we coudl be a little more descriptive with this scaladoc?
Perhaps This forms the root of the type system for Chisel's own types, and is the type needed to create IO, Wire, Reg, etc ?

Reply:
I took a crack at adding a more complete description. I didn't go so far as to enumerate the concrete subclasses, but I added more explanation about what IsBindable is.